numbers/compare.go

```diff
@@ -0,0 +1,16 @@
+package numbers
+
+import ()
+
+// EqualSliceInt is a function what will check two slices of int if the values are the same.
```

Comment:
Might be good to mention that order matters

Reply:
Added that into the GoDocs